libass/ass_parse.c

```diff
@@ -651,7 +652,17 @@ char *parse_tags(ASS_Renderer *render_priv, char *p, char *end, double pwr)
                 k = pow(((double) (t - t1)) / delta_t, accel);
             }
             p = args[cnt].start;
-            p = parse_tags(render_priv, p, args[cnt].end, k);    // maybe k*pwr ? no, specs forbid nested \t's
+            if (args[cnt].end < end)
+                p = parse_tags(render_priv, p, args[cnt].end, k);
+            else {
```

Comment:
(Minor nag: personally I'd prefer adding redundant `{` / `}` to the if body.)

Reply:
OK.